utils.py

```diff
@@ -62,8 +64,8 @@ def create_enum(*sequential, **names):
     Returns:
         dict. Dictionary containing the enumerated constants.
     """
-    enums = dict(zip(sequential, sequential), **names)
-    return type('Enum', (), enums)
+    enums = dict(list(python_utils.ZIP(sequential, sequential)), **names)
```

Comment:
`dict` constructor doesn't require a list as input, just return the result of ZIP.

Reply:
Done